### ISSUE TITLE
[Bug]: Diablo 2 Resurrected won't start

### ISSUE THREAD

**Issue by 4tuneTeller** [Whisky-App/Whisky#585]
### Description

Was able to install Battle.net, but when I start the game I get a blank screen and nothing happens. Tried turning MSync and DXVK on/off, but that didn't change anything. I'm trying to run it on MacBook Air M2 with 16 GB RAM

### Steps to reproduce

1. Install and launch Battle.net
2. Install D2R
3. Try to launch it
4. A black window with white stripe on the right appears but then nothings happens.

### Expected behaviour

The game launches and plays.

### Logs

```bash
Whisky Version: 2.1.1
Date: 02.11.2023, 15:12:18
macOS Version: Version 14.1 (Build 23B74)
Bottle Name: Battle.net
Wine Version: 7.7.0
Windows Version: win10
Bottle URL: /Volumes/ExtSSD/Whisky Bottles/D1E6ABBB-BBBE-4A7B-B89F-2217717790C8

Arguments: start /unix /Volumes/ExtSSD/Whisky Bottles/D1E6ABBB-BBBE-4A7B-B89F-2217717790C8/drive_c/Program Files (x86)/Battle.net/Battle.net Launcher.exe 

Launched Wine (1231)
msync: bootstrapped mach port on wine-30b4e-msync.
msync: up and running.
0068:err:module:open_builtin_so_file failed to load .so lib "/Users/timurnasyrov/Library/Application Support/com.isaacmarovitz.Whisky/Libraries/Wine/lib/wine/x86_64-unix/wineusb.sys.so"
0068:err:ntoskrnl:ZwLoadDriver failed to create driver L"\\Registry\\Machine\\System\\CurrentControlSet\\Services\\wineusb": c0000142
	VK_KHR_multiview v1
		VK_KHR_portability_subset v1
		VK_KHR_push_descriptor v2
		VK_KHR_relaxed_block_layout v1
		VK_KHR_sampler_mirror_clamp_to_edge v3
		VK_KHR_sampler_ycbcr_conversion v14
		VK_KHR_separate_depth_stencil_layouts v1
		VK_KHR_shader_draw_parameters v1
		VK_KHR_shader_float_controls v4
		VK_KHR_shader_float16_int8 v1
		VK_KHR_shader_subgroup_extended_types v1
		VK_KHR_spirv_1_4 v1
		VK_KHR_storage_buffer_storage_class v1
		VK_KHR_surface v25
		VK_KHR_swapchain v70
		VK_KHR_swapchain_mutable_format v1
		VK_KHR_timeline_semaphore v2
		VK_KHR_uniform_buffer_standard_layout v1
		VK_KHR_variable_pointers v1
		VK_EXT_buffer_device_address v2
		VK_EXT_debug_marker v4
		VK_EXT_debug_report v10
		VK_EXT_debug_utils v2
		VK_EXT_descriptor_indexing v2
		VK_EXT_external_memory_host v1
		VK_EXT_fragment_shader_interlock v1
		VK_EXT_hdr_metadata v2
		VK_EXT_host_query_reset v1
		VK_EXT_image_robustness v1
		VK_EXT_inline_uniform_block v1
		VK_EXT_memory_budget v1
		VK_EXT_metal_objects v1
		VK_EXT_metal_surface v1
		VK_EXT_pipeline_creation_cache_control v3
		VK_EXT_post_depth_coverage v1
		VK_EXT_private_data v1
		VK_EXT_robustness2 v1
		VK_EXT_sample_locations v1
		VK_EXT_scalar_block_layout v1
		VK_EXT_separate_stencil_usage v1
		VK_EXT_shader_atomic_float v1
		VK_EXT_shader_stencil_export v1
		VK_EXT_shader_viewport_index_layer v1
		VK_EXT_subgroup_size_control v2
		VK_EXT_surface_maintenance1 v1
		VK_EXT_swapchain_colorspace v4
		VK_EXT_swapchain_maintenance1 v1
		VK_EXT_texel_buffer_alignment v1
		VK_EXT_texture_compression_astc_hdr v1
		VK_EXT_vertex_attribute_divisor v3
		VK_AMD_gpu_shader_half_float v2
		VK_AMD_negative_viewport_height v1
		VK_AMD_shader_image_load_store_lod v1
		VK_AMD_shader_trinary_minmax v1
		VK_IMG_format_pvrtc v1
		VK_INTEL_shader_integer_functions2 v1
		VK_GOOGLE_display_timing v1
		VK_MVK_macos_surface v3
		VK_MVK_moltenvk v37
		VK_NV_fragment_shader_barycentric v1
		VK_NV_glsl_shader v1
01a0: thread_get_state failed on Apple Silicon - faking zero debug registers
0234:err:ntlm:ntlm_LsaApInitializePackage no NTLM support, expect problems
[mvk-info] GPU device:
		model: Apple M2
		type: Integrated
		vendorID: 0x106b
		deviceID: 0xe0103f0
		pipelineCacheUUID: DB8512A6-0E01-03F0-0000-000000000000
	supports the following Metal Versions, GPU's and Feature Sets:
		Metal Shading Language 3.0
		GPU Family Apple 8
		GPU Family Apple 7
		GPU Family Apple 6
		GPU Family Apple 5
		GPU Family Apple 4
		GPU Family Apple 3
		GPU Family Apple 2
		GPU Family Apple 1
		GPU Family Mac 2
		GPU Family Mac 1
		GPU Family Common 3
		GPU Family Common 2
		GPU Family Common 1
		macOS GPU Family 2 v1
		macOS GPU Family 1 v4
		macOS GPU Family 1 v3
		macOS GPU Family 1 v2
		macOS GPU Family 1 v1
	VK_KHR_multiview v1
		VK_KHR_portability_subset v1
		VK_KHR_push_descriptor v2
		VK_KHR_relaxed_block_layout v1
		VK_KHR_sampler_mirror_clamp_to_edge v3
		VK_KHR_sampler_ycbcr_conversion v14
		VK_KHR_separate_depth_stencil_layouts v1
		VK_KHR_shader_draw_parameters v1
		VK_KHR_shader_float_controls v4
		VK_KHR_shader_float16_int8 v1
		VK_KHR_shader_subgroup_extended_types v1
		VK_KHR_spirv_1_4 v1
		VK_KHR_storage_buffer_storage_class v1
		VK_KHR_surface v25
		VK_KHR_swapchain v70
		VK_KHR_swapchain_mutable_format v1
		VK_KHR_timeline_semaphore v2
		VK_KHR_uniform_buffer_standard_layout v1
		VK_KHR_variable_pointers v1
		VK_EXT_buffer_device_address v2
		VK_EXT_debug_marker v4
		VK_EXT_debug_report v10
		VK_EXT_debug_utils v2
		VK_EXT_descriptor_indexing v2
		VK_EXT_external_memory_host v1
		VK_EXT_fragment_shader_interlock v1
		VK_EXT_hdr_metadata v2
		VK_EXT_host_query_reset v1
		VK_EXT_image_robustness v1
		VK_EXT_inline_uniform_block v1
		VK_EXT_memory_budget v1
		VK_EXT_metal_objects v1
		VK_EXT_metal_surface v1
		VK_EXT_pipeline_creation_cache_control v3
		VK_EXT_post_depth_coverage v1
		VK_EXT_private_data v1
		VK_EXT_robustness2 v1
		VK_EXT_sample_locations v1
		VK_EXT_scalar_block_layout v1
		VK_EXT_separate_stencil_usage v1
		VK_EXT_shader_atomic_float v1
		VK_EXT_shader_stencil_export v1
		VK_EXT_shader_viewport_index_layer v1
		VK_EXT_subgroup_size_control v2
		VK_EXT_surface_maintenance1 v1
		VK_EXT_swapchain_colorspace v4
		VK_EXT_swapchain_maintenance1 v1
		VK_EXT_texel_buffer_alignment v1
		VK_EXT_texture_compression_astc_hdr v1
		VK_EXT_vertex_attribute_divisor v3
		VK_AMD_gpu_shader_half_float v2
		VK_AMD_negative_viewport_height v1
		VK_AMD_shader_image_load_store_lod v1
		VK_AMD_shader_trinary_minmax v1
		VK_IMG_format_pvrtc v1
		VK_INTEL_shader_integer_functions2 v1
		VK_GOOGLE_display_timing v1
		VK_MVK_macos_surface v3
		VK_MVK_moltenvk v37
		VK_NV_fragment_shader_barycentric v1
		VK_NV_glsl_shader v1
0494:err:sync:RtlpWaitForCriticalSection section 0000000170066620 "/private/tmp/game-porting-toolkit-20231101-9918-16fagj4/wine/dlls/ntdll/loader.c: loader_section" wait timed out in thread 0494, blocked by 0498, retrying (60 sec)
```

### What version of Whisky are you using?

2.1.1

### What version of macOS are you using?

Sonoma (macOS 14)

### Issue Language

- [X] Yes my issue is written in English

**Comment by 4tuneTeller** [comment]
<img width="1470" alt="Screenshot 2023-11-02 at 15 03 30" src="https://github.com/Whisky-App/Whisky/assets/13902650/ba97a792-736e-4af6-a117-d78e63f05798">

**Comment by IsaacMarovitz** [comment]
Hey @4tuneTeller please try again in Whisky 2.1.2, with Enhacned Sync set to `ESync`.

**Comment by 4tuneTeller** [comment]
Thank you for reply @IsaacMarovitz ! I've updated Whisky to 2.1.2, set Enhanced Sync to ESync and tried again, but nothing changed unfortunately. And I still see in the logs that MSync is up and running for some reason. Here's an updated log and a screenshot of my bottle settings:
```
Whisky Version: 2.1.2
Date: 03.11.2023, 10:46:51
macOS Version: Version 14.1 (Build 23B74)
Bottle Name: Battle.net
Wine Version: 7.7.0
Windows Version: win10
Bottle URL: /Volumes/ExtSSD/Whisky Bottles/D1E6ABBB-BBBE-4A7B-B89F-2217717790C8

Arguments: start /unix /Volumes/ExtSSD/Whisky Bottles/D1E6ABBB-BBBE-4A7B-B89F-2217717790C8/drive_c/Program Files (x86)/Battle.net/Battle.net Launcher.exe 

Launched Wine (2324)
msync: bootstrapped mach port on wine-30b4e-msync.
msync: up and running.
0068:err:module:open_builtin_so_file failed to load .so lib "/Users/timurnasyrov/Library/Application Support/com.isaacmarovitz.Whisky/Libraries/Wine/lib/wine/x86_64-unix/wineusb.sys.so"
0068:err:ntoskrnl:ZwLoadDriver failed to create driver L"\\Registry\\Machine\\System\\CurrentControlSet\\Services\\wineusb": c0000142
00e0: thread_get_state failed on Apple Silicon - faking zero debug registers
0160:err:winediag:wined3d_adapter_create Using the Vulkan renderer for d3d10/11 applications.
01a8: thread_get_state failed on Apple Silicon - faking zero debug registers
01e8: thread_get_state failed on Apple Silicon - faking zero debug registers
[mvk-info] MoltenVK version 1.2.3, supporting Vulkan version 1.2.243.
	The following 89 Vulkan extensions are supported:
		VK_KHR_16bit_storage v1
		VK_KHR_8bit_storage v1
		VK_KHR_bind_memory2 v1
		VK_KHR_buffer_device_address v1
		VK_KHR_copy_commands2 v1
		VK_KHR_create_renderpass2 v1
		VK_KHR_dedicated_allocation v3
		VK_KHR_depth_stencil_resolve v1
		VK_KHR_descriptor_update_template v1
		VK_KHR_device_group v4
		VK_KHR_device_group_creation v1
		VK_KHR_driver_properties v1
		VK_KHR_dynamic_rendering v1
		VK_KHR_external_fence v1
		VK_KHR_external_fence_capabilities v1
		VK_KHR_external_memory v1
		VK_KHR_external_memory_capabilities v1
		VK_KHR_external_semaphore v1
		VK_KHR_external_semaphore_capabilities v1
		VK_KHR_fragment_shader_barycentric v1
		VK_KHR_get_memory_requirements2 v1
		VK_KHR_get_physical_device_properties2 v2
		VK_KHR_get_surface_capabilities2 v1
		VK_KHR_imageless_framebuffer v1
		VK_KHR_image_format_list v1
		VK_KHR_maintenance1 v2
		VK_KHR_maintenance2 v1
		VK_KHR_maintenance3 v1
		VK_KHR_multiview v1
		VK_KHR_portability_subset v1
		VK_KHR_push_descriptor v2
		VK_KHR_relaxed_block_layout v1
		VK_KHR_sampler_mirror_clamp_to_edge v3
		VK_KHR_sampler_ycbcr_conversion v14
		VK_KHR_separate_depth_stencil_layouts v1
		VK_KHR_shader_draw_parameters v1
		VK_KHR_shader_float_controls v4
		VK_KHR_shader_float16_int8 v1
		VK_KHR_shader_subgroup_extended_types v1
		VK_KHR_spirv_1_4 v1
		VK_KHR_storage_buffer_storage_class v1
		VK_KHR_surface v25
		VK_KHR_swapchain v70
		VK_KHR_swapchain_mutable_format v1
		VK_KHR_timeline_semaphore v2
		VK_KHR_uniform_buffer_standard_layout v1
		VK_KHR_variable_pointers v1
		VK_EXT_buffer_device_address v2
		VK_EXT_debug_marker v4
		VK_EXT_debug_report v10
		VK_EXT_debug_utils v2
		VK_EXT_descriptor_indexing v2
		VK_EXT_external_memory_host v1
		VK_EXT_fragment_shader_interlock v1
		VK_EXT_hdr_metadata v2
		VK_EXT_host_query_reset v1
		VK_EXT_image_robustness v1
		VK_EXT_inline_uniform_block v1
		VK_EXT_memory_budget v1
		VK_EXT_metal_objects v1
		VK_EXT_metal_surface v1
		VK_EXT_pipeline_creation_cache_control v3
		VK_EXT_post_depth_coverage v1
		VK_EXT_private_data v1
		VK_EXT_robustness2 v1
		VK_EXT_sample_locations v1
		VK_EXT_scalar_block_layout v1
		VK_EXT_separate_stencil_usage v1
		VK_EXT_shader_atomic_float v1
		VK_EXT_shader_stencil_export v1
		VK_EXT_shader_viewport_index_layer v1
		VK_EXT_subgroup_size_control v2
		VK_EXT_surface_maintenance1 v1
		VK_EXT_swapchain_colorspace v4
		VK_EXT_swapchain_maintenance1 v1
		VK_EXT_texel_buffer_alignment v1
		VK_EXT_texture_compression_astc_hdr v1
		VK_EXT_vertex_attribute_divisor v3
		VK_AMD_gpu_shader_half_float v2
		VK_AMD_negative_viewport_height v1
		VK_AMD_shader_image_load_store_lod v1
		VK_AMD_shader_trinary_minmax v1
		VK_IMG_format_pvrtc v1
		VK_INTEL_shader_integer_functions2 v1
		VK_GOOGLE_display_timing v1
		VK_MVK_macos_surface v3
		VK_MVK_moltenvk v37
		VK_NV_fragment_shader_barycentric v1
		VK_NV_glsl_shader v1
[mvk-info] Created VkInstance for Vulkan version 1.1.0, as requested by app, with the following 2 Vulkan extensions enabled:
		VK_KHR_surface v25
		VK_EXT_metal_surface v1
02f4:err:ntlm:ntlm_LsaApInitializePackage no NTLM support, expect problems
0374: thread_get_state failed on Apple Silicon - faking zero debug registers
0394: thread_get_state failed on Apple Silicon - faking zero debug registers
0400: thread_get_state failed on Apple Silicon - faking zero debug registers
0458: thread_get_state failed on Apple Silicon - faking zero debug registers
048c: thread_get_state failed on Apple Silicon - faking zero debug registers
0494: thread_get_state failed on Apple Silicon - faking zero debug registers
04b0: thread_get_state failed on Apple Silicon - faking zero debug registers
04cc: thread_get_state failed on Apple Silicon - faking zero debug registers
[mvk-info] Created VkInstance for Vulkan version 1.1.0, as requested by app, with the following 2 Vulkan extensions enabled:
		VK_KHR_surface v25
		VK_EXT_metal_surface v1
054c: thread_get_state failed on Apple Silicon - faking zero debug registers
0570: thread_get_state failed on Apple Silicon - faking zero debug registers
0588: thread_get_state failed on Apple Silicon - faking zero debug registers
05b8: thread_get_state failed on Apple Silicon - faking zero debug registers
05c8: thread_get_state failed on Apple Silicon - faking zero debug registers
04e8:err:sync:RtlpWaitForCriticalSection section 0000000170066620 "/private/tmp/game-porting-toolkit-20231101-9918-16fagj4/wine/dlls/ntdll/loader.c: loader_section" wait timed out in thread 04e8, blocked by 0530, retrying (60 sec)
05f0: thread_get_state failed on Apple Silicon - faking zero debug registers
04e8:err:sync:RtlpWaitForCriticalSection section 0000000170066620 "/private/tmp/game-porting-toolkit-20231101-9918-16fagj4/wine/dlls/ntdll/loader.c: loader_section" wait timed out in thread 04e8, blocked by 0530, retrying (60 sec)
0600: thread_get_state failed on Apple Silicon - faking zero debug registers
```
<img width="756" alt="Screenshot 2023-11-03 at 10 53 24" src="https://github.com/Whisky-App/Whisky/assets/13902650/a545d220-7181-468b-9cf4-c7e9085cfb73">

**Comment by IsaacMarovitz** [comment]
> And I still see in the logs that MSync is up and running for some reason.
This has been fixed in 2.1.3, lemme know if that works for you

**Comment by 4tuneTeller** [comment]
@IsaacMarovitz Yes, it works now, thank you very much! :)
